src/app/credExplorer/pagerankTable/TableRow.test.js

```diff
@@ -8,33 +8,52 @@ import {COLUMNS} from "./sharedTestUtils";
 require("../../testUtil").configureEnzyme();
 
 describe("app/credExplorer/pagerankTable/TableRow", () => {
-  function example(depth: number = 3) {
+  function example() {
     return shallow(
       <TableRow
-        depth={depth}
+        depth={1}
+        indent={1}
         description={<span data-test-description={true} />}
         connectionProportion={0.5}
         score={133.7}
         children={<div data-test-children={true} />}
       />
     );
   }
-  it("has depth-based color", () => {
+  it("depth parameter changes the color", () => {
```

Comment:
Would be nice to test that this doesn’t also change the indentation, as
it does before the patch to the main code.